### PR TITLE
feat: mpl externs

### PR DESCRIPTION
**Pull request by ShubhamChaturvedi7**
_Issue #, if available:_

_Description of changes:_
NOTE: Scope of review here are the directories `AwsCryptographicMaterialProviders` and `.github` . Everything else is already in review in other PRs and diff would disappear from here when they are merged. You can still review them if you'd like.

_Squash/merge commit message, if applicable:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by seebees — APPROVED** [review]
LGTM, but it is a lot of Go :)

File: AwsCryptographicMaterialProviders/runtimes/go/ImplementationFromDafny-go/StormTrackingCMC/externs.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	for {
+		res := cmc.GetFromInner(input)
+		if res.IsFailure() {
+			return Companion_Default___.CreateGetCacheEntryFailure(res.Dtor_error().(AwsCryptographyMaterialProvidersTypes.Error))
+		} else if res.Dtor_value().(StormTracker.CacheState).Is_Full() {
+			return Companion_Default___.CreateGetCacheEntrySuccess(res.Dtor_value().(StormTracker.CacheState).Dtor_data())
+		} else if res.Dtor_value().(StormTracker.CacheState).Is_EmptyFetch() {
+			return Companion_Default___.CreateGetCacheEntryFailure(AwsCryptographyMaterialProvidersTypes.Companion_Error_.Create_EntryDoesNotExist_(_dafny.SeqOfChars([]_dafny.Char("Entry doesn't exists")...)))
+		} else {
+			time.Sleep(time.Duration(cmc.stormTracker.SleepMilli))
```
Comment (seebees) [inline AwsCryptographicMaterialProviders/runtimes/go/ImplementationFromDafny-go/StormTrackingCMC/externs.go:56]: see: https://github.com/aws/aws-cryptographic-material-providers-library/blob/main/AwsCryptographicMaterialProviders/runtimes/java/src/main/java/software/amazon/cryptography/internaldafny/StormTrackingCMC/StormTrackingCMC.java#L112-L132

This needs to keep track of how long it is sleeping. At a minimum, it can be an unbounded wait :)

File: StandardLibrary/runtimes/go/ImplementationFromDafny-go/Time_/externs.go [hunk 2]
```diff
@@ -18,7 +18,7 @@ func (CompanionStruct_Default___) CurrentRelativeTime() int64 {
 	return CurrentRelativeTime()
 }
 func CurrentRelativeTime() int64 {
-	return int64(time.Now().Second())
+	return int64(time.Now().Unix())
```
Comment (seebees) [inline StandardLibrary/runtimes/go/ImplementationFromDafny-go/Time_/externs.go:21]: Just curious why the change?

**Review by seebees — CHANGES_REQUESTED** [review]
Need to update the storm :)

File: AwsCryptographicMaterialProviders/runtimes/go/ImplementationFromDafny-go/StormTrackingCMC/externs.go (hunk 1 above)
Comment (ShubhamChaturvedi7) [reply AwsCryptographicMaterialProviders/runtimes/go/ImplementationFromDafny-go/StormTrackingCMC/externs.go]: Fixed.

File: StandardLibrary/runtimes/go/ImplementationFromDafny-go/Time_/externs.go (hunk 2 above)
Comment (ShubhamChaturvedi7) [reply StandardLibrary/runtimes/go/ImplementationFromDafny-go/Time_/externs.go:21]: Second gives the time in seconds since this process started. Unix gives the epoch time. It was a bug.

**Review by seebees — APPROVED** [review]
LGTM!
